packages/components/src/query-controls/README.md

```diff
@@ -66,24 +66,33 @@ The `QueryControls` component now supports multiple category selection, to repla
 const QUERY_DEFAULTS = {
 	orderBy: 'title',
 	order: 'asc',
-	selectedCategories: [ 1 ],
-	categories: [
+	selectedCategories: [
```

Comment:
This is changed to match the type that the 'Latest Posts' block passes to `<QueryControls>`:

<img width="807" alt="Screen Shot 2022-12-24 at 12 30 50 AM" src="https://user-images.githubusercontent.com/4063887/209424553-ed1c46e8-0588-456a-9f25-02cb14f584b0.png">